apm-agent-core/src/main/java/co/elastic/apm/impl/transaction/Span.java

```diff
@@ -84,6 +85,7 @@ private Span start(TraceContext parent, long epochMicros, boolean dropped) {
     }
 
     public Span startNoop() {
+        finished = false;
```

Comment:
Same comment goes here. This way, `finished` context can be limited to be AbstractSpan private